lib/bundler/resolver.rb

```diff
@@ -26,6 +26,9 @@ def message
           end.min_by(&:size)
           trees.reject! {|t| !maximal.include?(t.last) } if maximal
 
+          trees = trees.sort_by {|t| t.flatten.map(&:to_s) }
```

Comment:
We're doing another sort below, maybe we can combine the `sort_by`, `uniq`, and `sort_by` for perf reasons?